index.css

```diff
@@ -0,0 +1,97 @@
+/* General */
+
+body {
+    background-color: white;
+    height: 100%;
+    margin: 0px;
+    overflow: hidden;
+}
+
+p {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: white;
+    font-size: larger;
+}
+
+/* Header */
+
+.site-header {
+    background-color: black;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+#site-title-text {
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    color: white;
+    margin-left: 40vw;
```

Comment:
Nice! I wouldn't have thought to align the header like that, but it works surprisingly well.